localstack/services/events/events_starter.py

```diff
@@ -193,7 +193,7 @@ def events_handler_put_events(self):
                 'time': datetime.datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ'),
                 'region': self.region,
                 'resources': event.get('Resources', []),
-                'detail': json.loads(event.get('Detail')),
```

Comment:
Thanks for this PR @Josemaralves . I think we need to review the logic in here a bit - please note that `json.loads` takes a string as input and returns a `dict`, whereas `json_safe` takes a `dict` as input, and returns another `dict` (a copy of the input) which is safe for JSON dumping.

Is this the intention here, or can you please share the details of the error you're facing? Thanks